src/model.rs

```diff
@@ -159,6 +164,9 @@ pub enum StakeCredential {
 
 #[derive(Serialize, Deserialize, Debug, Clone, PartialEq, Eq, PartialOrd, Ord)]
 pub struct BlockRecord {
+    #[serde(serialize_with = "serialize_era")]
+    #[serde(deserialize_with = "deserialize_era")]
+    pub era: Option<Era>,
```

Comment:
@scarmuega is this what you meant? I also added a new config to mapper for it. Let me know if this is too much